src/routes/Apps/App/Dashboard.css

```diff
@@ -0,0 +1,4 @@
+.cl-dashboard-retry {
```

Comment:
Any reason not to re-use .cl-screen-reader?

Reply:
Yes, `.cl-screen-reader` came after this PR. It was not available to use.